spec/services/catalog/add_to_order_spec.rb

```diff
@@ -52,17 +52,19 @@
     end
 
     it 'can recreate the request from the context' do
-      item = nil
-      Insights::API::Common::Request.with_request(request) do
-        item = subject.order_item
-      end
+      item = Insights::API::Common::Request.with_request(request) { subject.order_item }
```

Comment:
This is just a small spec refactoring so we do not have to do `item = nil` before the block.  Not needed for this PR but felt it was a small enough change to include.  Easy to drop if we want to go without.